Docker.DotNet.X509/project.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "1.2.2",
+  "description": "X509Certificate and WebRequestHandler wrapper for Docker.DotNet",
+  "dependencies": {
+    "Docker.DotNet": "1.2.2"
+  },
+  "frameworks": {
+    "net46": {
```

Comment:
I just want to point out that this is full framework only. We can't do mutual-auth with certs with CoreCLR's HttpClientHandler implementation today.